### ISSUE TITLE
Can't switch to light mode

### ISSUE THREAD

**Issue by Jaisu-1** [matsuyoshi30/harbor#48]
I can't seem to switch to light mode in my settings. Are the light mode and dark mode versions two different projects?

**Comment by Jaisu-1** [comment]
For information, I am currently running on Ubuntu , maybe detecting system preference from Ubuntu is broken and is hence unable to switch to Light mode. Please look into this, I love this theme and would like to use it :/

**Comment by HirokiSAKABE** [comment]
The workaround: You can disable dark mode theme in [head.html](https://github.com/matsuyoshi30/harbor/blob/master/layouts/partials/head.html)

```diff
- <link id="dark-mode-theme" rel="stylesheet" href="{{ .Site.BaseURL }}css/dark.css" />
+ <link disabled id="dark-mode-theme" rel="stylesheet" href="{{ .Site.BaseURL }}css/dark.css" />
```

**Comment by Jaisu-1** [comment]
Is there no way to be able to toggle between the 2 themes?

**Comment by matsuyoshi30** [comment]
@Jaisu-1 Thanks reporting this. @HirokiSAKABE solved this problem by #49, so you can toggle between light and dark mode after updating the theme.

@HirokiSAKABE Thank you for great PR!
